routes/api.php

```diff
@@ -2,10 +2,17 @@
 
 use App\Http\Controllers\Api\UserController;
 use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\ImageController;
+use App\Http\Controllers\AuthorController;
+use App\Http\Controllers\CategoryController;
+
 
 Route::post('/auth/login', [UserController::class, 'login']);
+Route::apiResource('category', CategoryController::class);
+
+Route::post('/upload-icon', [ImageController::class, 'uploadIcon']);
```

Comment:
@Geri-PG Add {category} in the route name once you update the function for uploading the icon